content/ja/gethelp.md

```diff
@@ -3,19 +3,19 @@ title: サポートを得る方法
 sidebar: false
 ---
 
-**ユーザーからの質問:** ユーザーからの質問に対して回答を得る最も良い方法は、[StackOverflow](http://stackoverflow.com/questions/tagged/numpy)に質問を投稿することです。すでに数千ものユーザーからの回答を見ることができます。  規模は小さいですが、下記のような質問をする場所もあります: [IRC](https://webchat.freenode.net/?channels=%23numpy), [Gitter](https://gitter.im/numpy/numpy), [Reddit](https://www.reddit.com/r/Numpy/)。 私たちはこれらのサイトを定期的に確認して、直接質問に答えるようにしていますが、質問の数は膨大です。
+**ユーザーからの質問:** ユーザーからの質問に対して回答を得る最も良い方法は、[StackOverflow](http://stackoverflow.com/questions/tagged/numpy)に質問を投稿することです。すでに数千ものユーザーからの回答を見ることができます。  規模は小さいですが、下記のような質問をする場所もあります: [IRC](https://webchat.freenode.net/?channels=%23numpy)、 [Gitter](https://gitter.im/numpy/numpy)、 [Reddit](https://www.reddit.com/r/NumPy/)。 私たちはこれらのサイトを定期的に確認して、直接質問に答えるようにしていますが、質問の数は膨大です。
```

Comment:
maybe the URL should not converted? It works though.

Reply:
Nice catch! It was certainly because of a mistake.